x-pack/plugins/aiops/public/components/document_count_content/document_count_content/document_count_content.tsx

```diff
@@ -45,19 +45,25 @@ export const DocumentCountContent: FC<DocumentCountContentProps> = ({
   const { documentStats } = useAppSelector((s) => s.logRateAnalysis);
   const { sampleProbability, totalCount, documentCountStats } = documentStats;
 
+  const isCasesEmbedding = embeddingOrigin === AIOPS_EMBEDDABLE_ORIGIN.CASES;
```

Comment:
Could the Run Analysis and Reset buttons be hidden rather than disabled when inside a case?

<img width="554" alt="Screenshot 2024-11-27 at 11 19 48" src="https://github.com/user-attachments/assets/f7699ab7-cd3c-4bc5-b282-21b2474ffcff">

Reply:
Done in: [#0713825](https://github.com/elastic/kibana/pull/201549/commits/071382510445641e894c88fb2a3b9ea9e596cfa4)